WalletWasabi.Fluent/ViewModels/HelpAndSupport/LinkViewModel.cs

```diff
@@ -0,0 +1,31 @@
+using System.IO;
+using System.Windows.Input;
+using Avalonia;
+using ReactiveUI;
+
+namespace WalletWasabi.Fluent.ViewModels.HelpAndSupport
+{
+	public partial class LinkViewModel : ViewModelBase
```

Comment:
is the `partial` needed?

Reply:
Not really necessary when you don't use [AutoNotify] or other source generator stuff

Reply:
Pushed changes